runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorkerTest.java

```diff
@@ -799,6 +799,7 @@ public void testHotKeyLogging() throws Exception {
           makeInput(i, TimeUnit.MILLISECONDS.toMicros(i), "key", DEFAULT_SHARDING_KEY));
     }
 
+    @SuppressWarnings("unused")
     Map<Long, Windmill.WorkItemCommitRequest> result = server.waitForAndGetCommits(numIters);
```

Comment:
Can we just do this instead? (same for the other suppression in this file)
```suggestion
    server.waitForAndGetCommits(numIters);
```

Reply:
Yes, done